src/components/ContributeAs.js

```diff
@@ -0,0 +1,249 @@
+    const showSearch = Object.keys(profiles).length >= 5 || state.search;
+
+    return (
+      <StyledCard maxWidth={500}>
+        {showSearch && (
+          <Container display="flex" borderBottom="1px solid" borderColor="black.200" px={4} py={1} alignItems="center">
+            <SearchIcon size="16" />
+            <StyledInput
+              bare
+              type="text"
+              fontSize="Paragraph"
+              lineHeight="Paragraph"
+              placeholder="Filter by name..."
+              onChange={onSearch}
+            />
+          </Container>
+        )}
+        <StyledRadioList {...fieldProps} options={options} onChange={onChange} defaultValue={firstProfile}>
+          {({ key, value, radio, checked, index }) => (
+            <Container
+              display="flex"
+              alignItems="center"
+              px={4}
+              py={3}
+              borderBottom={lastIndex !== index ? '1px solid' : 'none'}
+              color={key === 'anonymous' && checked ? 'white.full' : 'black.900'}
+              bg={key === 'anonymous' && checked ? 'black.900' : 'white.full'}
+              borderColor="black.200"
+              flexWrap="wrap"
+            >
+              <Box as="span" mr={3}>
+                {radio}
+              </Box>
+              {value.type === 'USER' ? (
+                <Avatar src={value.image} type={value.type} size="3.6rem" name={value.name} />
+              ) : (
+                <Logo src={value.image} type={value.type} height="3.6rem" name={value.name} />
+              )}
+              <Flex flexDirection="column" ml={2}>
+                <P color="inherit" fontWeight={value.type ? 600 : 500}>
+                  {value.name}
+                </P>
+                {value.type && (
+                  <P fontSize="Caption" lineHeight="Caption" color="black.500">
+                    {key === '0' ? `Personal account - ${value.email}` : capitalize(value.type)}
+                  </P>
+                )}
+              </Flex>
+              {key === 'new-org' && checked && (
+                <Container as="fieldset" border="none" width={1} py={3} onChange={onFieldChange}>
+                  <Box mb={3}>
+                    <StyledInputField label="Org Name" htmlFor="orgName" error={getFieldError('orgName')}>
+                      {inputProps => (
+                        <StyledInput
+                          {...inputProps}
+                          {...getFieldProps(inputProps.name)}
+                          placeholder="i.e. AirBnb, Women Who Code"
+                          required
+                        />
+                      )}
+                    </StyledInputField>
+                  </Box>
+
+                  <Box mb={3}>
+                    <StyledInputField label="Website" htmlFor="website" error={getFieldError('website')}>
+                      {inputProps => <StyledInput {...inputProps} {...getFieldProps(inputProps.name)} type="url" />}
```

Comment:
Should'nt this be a `StyledInputGroup` with prepend="https://"?

Reply:
I wasn't sure about that because I don't want to confuse people who may have a site that still uses `http://` and the `url` input will provide feedback on the format already.